tests/test_distributions.py

```diff
@@ -333,6 +498,25 @@ def test_eq_ne_hash() -> None:
         assert not d != d_copy
         assert hash(d) == hash(d_copy)
 
+    # Different field values.
+    di0 = distributions.FloatDistribution(low=1, high=2)
+    di1 = distributions.FloatDistribution(low=1, high=3)
+    assert di0 != di1
+    assert not di0 == di1
```

Comment:
[question] I'd like to know why it checks `not di0 == di1` in addition to `di0 != di1` (is it not equivalent?).

Reply:
Thank you for the comments, sorry I mostly copied those from existing tests. I think those 2 statements are equivalent.

Reply:
Sorry for pointing out something that wasn't your change. But I think it would be nice if we can cleanup tests as well.
Thank you for the quick updates as always!

Reply:
No problem, reflected those changes to existing checks as well. Thank you.